app/ocr.py

```diff
@@ -506,13 +507,20 @@ def get_matches(input_list, topic, full_text):
         violent_list = ['abduct', 'abuse', 'assassinate', 'assault', 'coerce',
                         'exploit', 'fear', 'harm', 'hurt', 'kidnap', 'kill',
                         'murder', 'persecute', 'rape', 'scare', 'shoot',
-                        'suffer', 'threat', 'torture']
+                        'suffer', 'threat', 'torture', 'attack', 'displace',
+                        'massacre', 'genocide']
+
         family_list = ['child', 'daughter', 'family', 'husband', 'parent',
-                       'partner', 'son', 'wife', 'woman']
+                       'partner', 'son', 'wife', 'woman', 'spouse', 'father',
+                       'aunt', 'uncle']
+
         gender_list = ['fgm', 'gay', 'gender', 'homosexual', 'homosexuality',
-                       'lesbian', 'lgbt', 'lgbtq', 'lgbtqia',
-                       'queer', 'sexuality', 'transgender']
-        gang_list = ['cartel', 'gang', 'militia']
+                       'lesbian', 'lgbt', 'lgbtq', 'lgbtqia', 'queer',
+                       'sexuality', 'transgender', 'bisexual', 'lesbian']
+
+        gang_list = ['cartel', 'gang', 'militia', 'isis', 'taliban', 'al-qaeda',
+                     'al qaeda', 'daesh', 'shia', 'sunni', 'isil', 'boko haram', 'jihadists',
+                     'uighyers', 'kulunas', 'mara salvatrucha 13', 'ms 13']
```

Comment:
WOW. Great job drumming up a slew of new keywords we can scrape for - this must increase our accuracy, I'm sure!